package.json

```diff
@@ -3,7 +3,7 @@
     "@atlaskit/tree": "^8.6.3",
     "@cquiroz/react-moon": "^2.0.2",
     "@floating-ui/react-dom-interactions": "^0.10.3",
-    "@tanstack/react-table": "^8.7.0",
+    "@tanstack/react-table": "^8.5.22",
```

Comment:
You're downgrading react-table?

Reply:
Yup, this is the version we are using in the subproject and the one used by ST.

I tried bumping the one in the subproject but then the code generated by ST introduces a lot of breaking changes. We should investigate that in another PR.